code/_onclick/hud/plane_master.dm

```diff
@@ -27,7 +27,7 @@
 /atom/movable/screen/plane_master/openspace/Initialize(mapload)
 	. = ..()
 	filters += filter(type="alpha", render_source=FIELD_OF_VISION_RENDER_TARGET, flags=MASK_INVERSE)
-
+	filters += filter(type="alpha", render_source = LIGHTING_RENDER_TARGET, flags = MASK_INVERSE)
```

Comment:
The filter above this one is not present in tg, I'm not sure if having two alpha filters on this plane_master is an issue, but it should hopefully not be, and looks fine in testing.